### ISSUE TITLE
Docker container does not start

### ISSUE THREAD

**Issue by Gattag** [denverquane/amongusdiscord#43]
```
version: '3.3'
services:
  bot:
    image: denverquane/amongusdiscord
    ports:
      - 8123:8123
    environment:
      - DISCORD_BOT_TOKEN=<my token>
    restart: unless-stopped
```

```
Program exited with the following error:
error loading environment file; you need a file named final.env, final.txt, or final.env.txt
This window will automatically terminate in 30 seconds
```

To solve the issue in my case I did this
```
version: '3.3'
services:
  bot:
    image: denverquane/amongusdiscord
    ports:
      - 8123:8123
    environment:
      - DISCORD_BOT_TOKEN=<my token>
    volumes:
      - type: bind
        source: ./final.txt
        target: /go/src/app/final.txt
    restart: unless-stopped
```
`final.txt` contains `DISCORD_BOT_TOKEN=<my token>`

**Comment by Gattag** [comment]
I also ran `docker run -p 8123:8123 -e DISCORD_BOT_TOKEN=<my token> denverquane/amongusdiscord` but it had the same issue as expected

**Comment by denverquane** [comment]
Sorry about that! Fixed now in commit a80524f, program will continue even with no env file; assuming the DISCORD_BOT_TOKEN is provided
